doc/integrator/ngeo.rst.tmpl

```diff
@@ -97,8 +104,8 @@ The sub section is the interface name, and after that we have:
 * ``extends``: Interface for which the ``constants``, ``dynamic_constants``, ``static`` and ``routes`` shall
     be extended. These values will be updated with the interface configuration
     (but can not be removed via interface configuration).
-* ``constants``: Directly define an ``angular`` constant in the vars file.
-* ``dynamic_constants``: Define an ``angular`` constant from a dynamic values.
+* ``constants``: Directly define a constant in the vars file.
+* ``dynamic_constants``: Define a constant from a dynamic values.
```

Comment:
Later in this section, we point to a doc to know every possible constant. But Some are missing like "gmfCustomCSS". That's because that's NOT an AngularJS constant. should we create a Jira Ticket to publish the ngeo `srcapi/model/config` doc ?

Reply:
Effectively, The JIRA ticket already exists: GSGMF-1674, you can edit it to add some previsions :-)

Reply:
With the pull request https://github.com/camptocamp/ngeo/pull/7707 the link to the configuration can be updated to
```
https://camptocamp.github.io/ngeo/{version}/apidoc/modules/srcapi_store_config.html#Configuration
```
(see: https://camptocamp.github.io/ngeo/documentation-GSGMF-1674/apidoc/modules/srcapi_store_config.html#Configuration)

Reply:
Nice, thanks !